internal/repository/repository.go

```diff
@@ -741,15 +741,10 @@ func (r *Repository) List(ctx context.Context, t restic.FileType, fn func(restic
 
 // ListPack returns the list of blobs saved in the pack id and the length of
 // the file as stored in the backend.
-func (r *Repository) ListPack(ctx context.Context, id restic.ID, size int64) ([]restic.Blob, int64, error) {
+func (r *Repository) ListPack(ctx context.Context, id restic.ID, size int64) ([]restic.Blob, uint32, error) {
```

Comment:
What's going on here? Before, `ListPack()` returned its parameter `size` (which is a bit pointless), afterwards the size of the header is returned? If that's correct, we should amend the docstring here and in the `restic.Repository` interface.

Reply:
Right - I chose to exchange this as this `size` was - no wonder - not used.
The comment is changed, sorry for the confusion!